src/Microsoft.Health.Fhir.Liquid.Converter.UnitTests/Processors/ProcessorTests.cs

```diff
@@ -362,5 +366,45 @@ public void GivenJObjectInput_WhenConvertWithJsonProcessor_CorrectResultShouldBe
             var result = processor.Convert(testData, "ExamplePatient", templateProvider);
             Assert.True(JToken.DeepEquals(JObject.Parse(_jsonExpectData), JToken.Parse(result)));
         }
+
+        [Fact]
+        public void GivenJObjectInput_WhenConvertWithJsonToHl7v2Processor_CorrectResultShouldBeReturned()
+        {
+            var processor = new JsonToHl7v2Processor(_processorSettings);
+            var templateProvider = new TemplateProvider(TestConstants.TestTemplateDirectory, DataType.Json);
+            var testData = JObject.Parse(_fhirBundleTestData);
+            var result = processor.Convert(testData, "BundleToHl7v2", templateProvider);
+            Assert.Equal(_fhirBundleExpectedData, result);
+        }
+
+        [Fact]
```

Comment:
nit suggestion: It seems like the test setup and assertion is similar for these 3 tests where exception is expected. Maybe making this a Theory test and passing the file name and expected error code will help reduce the # of lines, but if you want to have separate test case names then fine to leave as is.